### DISCUSSION TITLE
po-datepicker formatos e exibição de datas

### DISCUSSION THREAD

**Discussion by FerriDaniel** [po-ui/po-angular#954] [Angular]
Olá.
Estou desenvolvendo um projeto onde tenho dois campos de data com formatos diferente, e não encontrei na documentação algo que atendesse o requisito.
Campo 1: _MM/YYYY_
Campo 2: _YYYY_
Também não encontrei onde eu inicializo a visualização da seleção de datas, no caso seria iniciado com seleção de Ano.

Como poderia implementar estes campos utilizando os componentes?

**Comment by jhonyeduardo** [comment]
Boa tarde @FerriDaniel 

Não temos essa funcionalidade habilitada no componente, porém temos ela em nosso backlog para implementação futura.
